roles/katello_devel/meta/main.yml

```diff
@@ -20,6 +20,6 @@ dependencies:
     foreman_installer_options_internal_use_only:
       - "{{ '--katello-devel-scl-ruby=' +  ruby_scl_version if ansible_distribution_major_version == '7' else '' }}"
       - "--katello-devel-admin-password {{ foreman_installer_admin_password }}"
-      - "--katello-devel-github-username '{{ katello_devel_github_username }}'"
+      - "--katello-devel-github-username '{{ katello_devel_github_username | default('') }}'"
```

Comment:
There is an option to omit things. Perhaps the whole option should not be given if it's empty?

Reply:
@ekohl the `default(omit)` option doesn't work here at least in terms of omitting the entire entry from the internal_use_only array. do you have a clean approach to do that? if not, I'm personally OK with passing an empty string - it doesn't cause any problems

Reply:
A few lines above we have a similar pattern for `ruby_scl_version`. Perhaps that's a cleaner approach?

Reply:
got it. updated!